.github/workflows/deploy-to-production.yml

```diff
@@ -26,7 +26,6 @@ jobs:
           # setting CI to false is temporary, while we still have lint errors
           CI: false
         run: npm run build
-      - run: npm audit --production
```

Comment:
audit is already run before merge, as we can catch things before merge this felt a bit unnecessary.

Reply:
I'd like to keep this in - or actually to replace it with `run check:audit`, so at least we can suppress reviewed false positives.

Agree it's a bit unnecessary, and somewhat annoying due to the high rate of false positives on `npm audit` largely caused by the fact the ReDOS issues are largely irrelevant to a client-side web app. But the security team here have requested this in the past, just in case a true positive ever arose, so let's keep it in.